app/js/components/orderEntry/OrderHeader.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import IconButton from '../button/IconButton';
 
 const OrderHeader = ({ status, orderable }) => (
   <Fragment>
     <th>{orderable}</th>
     <th>{status}</th>
-    <th>
-      <i className="icon-edit" />
-      <i className="icon-delete" />
+    <th className="draft-action-btns">
+      <IconButton
```

Comment:
`className="draft-action-btns"`: This class name makes it seem like the action buttons in this component are in a draft container but that is not the case, so consider refactoring the class naming.